libs/ui-watt/src/lib/components/shell/nav-list/watt-nav-list-item.component.ts

```diff
@@ -14,34 +14,55 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 import {
   ChangeDetectionStrategy,
   Component,
   Input,
   NgModule,
 } from '@angular/core';
-import { RouterModule } from '@angular/router';
-import { MatRippleModule } from '@angular/material/core';
+
+import { CommonModule } from '@angular/common';
 import { MatListModule } from '@angular/material/list';
+import { MatRippleModule } from '@angular/material/core';
+import { RouterModule } from '@angular/router';
 
 @Component({
   changeDetection: ChangeDetectionStrategy.OnPush,
   selector: 'watt-nav-list-item',
-  template: `<a
-    mat-list-item
-    mat-ripple
-    [routerLink]="link"
-    routerLinkActive="active"
-    ><ng-content></ng-content
-  ></a>`,
+  template: `
+    <a
+      *ngIf="isExternal; else internalLink"
+      mat-list-item
+      mat-ripple
+      [href]="link"
+      [target]="target"
+      ><ng-container *ngTemplateOutlet="templateContent"></ng-container
+    ></a>
+
+    <ng-template #internalLink>
+      <a mat-list-item mat-ripple [routerLink]="link" routerLinkActive="active"
+        ><ng-container *ngTemplateOutlet="templateContent"></ng-container
+      ></a>
+    </ng-template>
+
+    <ng-template #templateContent>
+      <ng-content></ng-content>
+    </ng-template>
+  `,
 })
 export class WattNavListItemComponent {
   @Input() link: string | null = null;
+  @Input() target = '_self';
+
+  get isExternal(): boolean {
+    return this.link?.startsWith('https://') ?? false;
+  }
```

Comment:
nitpick: rename `isExternal` to `isExternalLink` to be more explicit

issue: links starting with `http://` (note missing `s`) will be treated as internal. Take that into consideration.

Reply:
Non-secure links should never be used 🙂

Reply:
+1 to nitpick

Reply:
Agree as long as we know that the external pages we're going to link to always use HTTPS. If that's the case, I'm happy to drop the comment. 👍